src/BloomBrowserUI/karma.conf.js

```diff
@@ -89,7 +89,7 @@ module.exports = function (config) {
         // - Safari (only Mac)
         // - PhantomJS
         // - IE (only Windows)
-        browsers: ['Firefox'],
+        browsers: ['Chrome'],
```

Comment:
No. But you can't debug the tests on Firefox. Something about their plug-in not working with Webstorm anymore.
I don't think it affects the running tests... maybe we should ask Phillip?